examples/speech_recognition/infer.py

```diff
@@ -186,7 +186,7 @@ def main(args):
     # Load ensemble
     logger.info("| loading model(s) from {}".format(args.path))
     models, criterions, _model_args = load_models_and_criterions(
-        args.path.split(":"),
+        args.path.split("::"),
```

Comment:
Can you replace this with `os.pathsep` instead: https://docs.python.org/3/library/os.html#os.pathsep